ADRs/RH/SIG-SRE/ADR-00005 SLO Lifecycle.md

```diff
@@ -0,0 +1,131 @@
+
+### Implementation Phase
+
+With an accepted proposal from the Research Phase, an owner can be found to implement the changes. Not all of these changes are necessarily made by software engineers. Certainly changes to instrument the service’s codebase is in the domain of the software engineer, but changing team policies and practices is more likely to be the domain of the engineering lead. Remember that the work isn’t intended to be wholly owned by the person to whom it is assigned, that person can and should work in concert with people who can assist. [Consider the RACI chart][raci-chart] which shows that more than just one role is involved; work as a team to accomplish the goals.
+
+The owner will likely be an engineer on the service's team who may work in concert with an appropriate SRE IC to affect changes to the service and related ecosystem.
+
+By way of example, some of these changes may include (but not limited to):
+
+* SLI (Service Level Indicator) instrumentation in the service
+  * Software changes
+* SLO Dashboard Changes
+  * Creation of SLO Dashboard
+  * Changes to existing SLO dashboard to incorporate new changes
+* Changes to team policies and practices to support the [Iteration Phase](#iteration-phase)
+
+### Iteration Phase
+
+Once the SLOs are adopted, the [service stakeholders][raci-chart] will enter the lifecycle loop where the team and service come to live with the SLO. Now the team uses the previous work to instrument and monitor the service. Sprint planning is influenced by the error budget. Existing SLOs are continually evaluated for needed changes, or even their retirement. Changes will want to kick off a new Research Phase to propose and understand the scope of changes.
+
+#### SLO Review
+
+Part of healthy SLO adoption is the periodic review of the service's SLO needs. These review tasks may include:
+
+* Do the service's SLOs still measure the right thing(s) to measure customer happiness and service reliability?
+* Are the SLOs' SLIs still appropriate? Are there newer, better (ex. less computationally expensive) indicators that should be used?
+* Is a SLO still needed? (ex. Does the service landscape or user experience still demand it? Has another SLO superseded it?)
+
+These steps can be used to evaluate the SLOs themselves, but there's also review on the targets set by the service's SLO(s):
+
+* Are the service's SLOs being met?
+  * If they're not, what are common reasons why not?
```

Comment:
And who's job is it to fix?  e.g. do we have upstream quality issues that could be caught by new tests?